src/scripts/gym/GymPokemon.ts

```diff
@@ -3,9 +3,10 @@ class GymPokemon {
     maxHealth: number;
     level: number;
     shiny: boolean;
+    shadow: boolean; // Currently don't do anything. Will be added later
```

Comment:
Should this not be a type of `ShadowStatus` instead of boolean?

Reply:
It was a boolean because i did not think purified ever made sense for trainer mons. But we will have an icon, so it might make sense at some point... I have changed it